pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+
+
+....................................................
+Before you submit a pull request, please make sure :
+
+- code contains tests relevant for the problem you are solving
```

Comment:
This item needs some clarification. Do we now expect all developers to add unit tests any time they make a PR? This is a nice idea in theory, but:
1. Nobody will actually do it
2. Unit tests can cause more harm than good (e.g. if they rely on unavailable input files, I can think of a few frequent offenders already in the repository)